odl/tomo/backends/astra_cpu.py

```diff
@@ -249,6 +249,65 @@ def astra_cpu_back_projector(proj_data, geometry, vol_space, out=None,
     return out
 
 
+class AstraCpu:
```

Comment:
I was thinking this might clash with the ASTRA naming for a _Projector_, which is either a forward projector or a backprojector, while our class does both. I now named this `<backend>Impl`. What do you think?

Reply:
Ah yes, that's fine. `Impl` sounds so obscure that nobody will dare to use these classes directly :wink: